qcore/timeseries.py

```diff
@@ -758,7 +795,7 @@ def __init__(self, bb_path):
         # read header - strings
         self.lf_dir, self.lf_vm, self.hf_file = np.fromfile(
             bbf, count=3, dtype="|S256"
-        ).astype(np.str_)
+        ).astype(np.unicode_)
 
```

Comment:
`np.unicode_` is an alias of `np.str_` [according to the docs](https://numpy.org/doc/1.25/reference/arrays.scalars.html#numpy.unicode_). Is there any reason to use that?

Reply:
This change somehow slipped in: https://github.com/ucgmsim/qcore/commit/20de500cc3e06758e6919056b182cfa8c4b21909